ambry-network/src/main/java/com.github.ambry.network/PlainTextTransmission.java

```diff
@@ -59,7 +59,7 @@ public boolean ready() {
   @Override
   public boolean read() throws IOException {
     if (!hasReceive()) {
-      networkReceive = new NetworkReceive(getConnectionId(), new BoundedByteBufferReceive(), time);
+      createNetworkReceive();
```

Comment:
prefer to have a return value for explicitly set.

Reply:
The networkReceive is a private field. createNetReceive is a private method. I suppose we can create it in the method. Is it the method name? Maybe I can change the method name to something like "initializeNetworkReceive"?